tests/tests_unit/test_api/test_assets.py

```diff
@@ -125,6 +141,24 @@ def test_list_root(self, mock_assets_response):
             calls[0].request.body
         )
 
+    def test_list_partitions(self, mock_assets_response):
+        res = ASSETS_API.list(partitions=3, limit=None)
+        assert isinstance(res, AssetList)
+        assert isinstance(res[0], Asset)
+        assert 3 == len(mock_assets_response.calls)
+        assert {"1/3", "2/3", "3/3"} == {jsgz_load(c.request.body)["partition"] for c in mock_assets_response.calls}
```

Comment:
When you are testing the generic behaviour for _list_partitioned(), it should be in a test in test_api_client.py.

Reply:
done + fixed bugs this revealed :+1: